README.md

```diff
@@ -2,6 +2,8 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
+[_Griffel (German): stylus, pen, slate pencil; an antiquated writing tool, used in ancient times, prior to the appearance of blackboards._](https://en.wiktionary.org/wiki/Griffel)
+
 Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```

Comment:
```suggestion
[_Griffel (German): stylus, pen, slate pencil; an antiquated writing tool, used in ancient times, prior to the appearance of blackboards._](https://en.wiktionary.org/wiki/Griffel)

Griffel ([_German_]((https://en.wiktionary.org/wiki/Griffel)) for stylus/pen) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```

![image](https://user-images.githubusercontent.com/14183168/154042908-ff18f7e6-1a0b-468c-9b2b-62eb3afe81aa.png)

- Not sure that it's the most important thing in README 🙃
- The suggestion IMO looks more accurate